exonum_client/client.py

```diff
@@ -233,6 +233,42 @@ def available_services(self) -> requests.Response:
         """
         return _get(_SYSTEM_URL.format(self.schema, self.hostname, self.public_api_port, "services"))
 
+
+    def get_instance_id_by_name(self, name: str) -> int:
+        """
+        Gets an instance id with provided name.
+
+        Example:
+
+        >>> id = client.get_instance_id_by_name("cryptocurrency")
+        >>> id
+        42
+
+        Parameters
+        ----------
+        name: Name of the service.
+
+        Returns
```

Comment:
If function raises an exception, it should be mentioned in the "Raises" section of the doccoment.